source/routing/raptor.cpp

```diff
@@ -602,6 +597,12 @@ void RAPTOR::raptor_loop(Visitor visitor, const type::AccessibiliteParams & acce
                             l_zone = it_st->local_traffic_zone;
                             continue_algorithm = true;
                             BOOST_ASSERT(! visitor.comp(workingDt, previous_dt));
+
+                            if (jpp.has_freq) {
```

Comment:
you must check if the vj has freq, jpp.has_freq says that there is at least one freq vj on the jp.